migrations/versions/201603171003_3d672dc5ee53_unify_lecture_style_ids.py

```diff
@@ -0,0 +1,49 @@
+"""Unify lecture style ids
+
+Revision ID: 3d672dc5ee53
+Revises: 29232c09e58a
+Create Date: 2016-03-17 10:03:17.857070
+"""
+
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = '3d672dc5ee53'
+down_revision = '29232c09e58a'
+
+
+def _move_theme(from_, to):
+    op.execute("""
+        UPDATE events.settings
+        SET value = '"%s"'::json
+        WHERE name = 'timetable_theme'
```

Comment:
I'd also add a `module = '...'` since the name by itself is not enough to uniquely identify a setting (even though it works since the name is unique in this case, it probably won't make use if the index we have)